sys-process/audit/audit-3.0.6.ebuild

```diff
@@ -144,14 +140,18 @@ multilib_src_install_all() {
 	# newinitd "${FILESDIR}"/auditd-init.d-2.4.3 auditd
 	# newconfd "${FILESDIR}"/auditd-conf.d-2.1.3 auditd
 
+	# Flatcar: install sample configuration
+	insinto /usr/share/auditd
+	doins "${FILESDIR}"/init.d/auditd.conf
```

Comment:
This path here is not yet correct: `FileNotFoundError: [Errno 2] No such file or directory: b'/build/amd64-usr/var/tmp/portage/sys-process/audit-3.0.6/files/init.d/auditd.conf'`
I think we need `WORKDIR` or `S` instead of `FILESDIR`, I'll have to give it a try

Reply:
Let me know if using `WORKDIR` works for you

Reply:
```suggestion
	doins "${S}"/init.d/auditd.conf
```
This here works